heron/proto/BUILD

```diff
@@ -135,61 +135,6 @@ proto_library(
     ],
 )
 
-java_library(
```

Comment:
I have already explained this in the PR: "Removed some unused protobuf build targets.".